changelog.d/308.breaking.rst

```diff
@@ -0,0 +1,15 @@
+The ``__repr__`` set by ``attrs`` no longer produces an ``AttributeError`` when
```

Comment:
[Semantic newlines](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) in documentation please.

Reply:
Huh, I'd never read that doc.  Cool.

Reply:
verified fixed